packages/block-library/src/heading/edit.js

```diff
@@ -47,8 +45,8 @@ function HeadingEdit( {
 		<>
 			<BlockControls>
 				<HeadingToolbar
-					minLevel={ 2 }
-					maxLevel={ 5 }
+					minLevel={ 1 }
+					maxLevel={ 7 }
```

Comment:
No, this is not a mistake. The `maxLevel` is actually exclusive rather than inclusive, so you have to set it to 7 if you want to include heading level 6. I thought that was weird, but I decided it would be best to address it in a separate PR, if it needs to be addressed at all. Does anyone else think this is weird?

Reply:
The fact that heading 1 was omitted from the toolbar was a requirement/recommendation from the a11y team to discourage its usage as it's generally used for post title in the templates. Is this something we want to reconsider?

Reply:
@youknowriad Some kind of warning would be helpful, similar to the color contrast warning (which can be used in the toolbar, as demonstrated by the Navigation block). No equivalent currently exists for heading levels except in the Document Outline.

Showing a duplicate control in the inspector with the discouraged options is not the right answer, in my opinion. We should be dealing with incorrect heading levels the same way we deal with low-contrast background/text color combinations: by warning the user when they choose a potentially poor option.